ethereum_history_api/circuits/lib/src/fixtures/paris/usdc/storage_proof.nr

```diff
@@ -0,0 +1,28 @@
+
+use crate::storage::{StorageProof, StorageProofsWithStorageRoot};
+use crate::misc::types::Bytes32;
+use crate::account::PROOF_LEN;
+
+global storage_root: Bytes32 = [236,249,35,40,236,79,19,252,192,40,24,249,103,163,215,123,232,167,225,17,214,124,198,236,16,58,87,150,142,162,11,197];
```

Comment:
Let's adopt a common convention about how do we represent long lists in those files.
For now we have this everywhere:
```rust
global var = [
    1, 2, 3, ...
];
```
but here we have this: 
```
global var = [1, 2, 3, ...];
```

I don't have a strong preference. I agree that the second one is more compact. But if we're changing it - let's change everywhere